solidity_adapter/src/test/mod.rs

```diff
@@ -36,7 +36,7 @@ impl TryFrom<&Path> for Test {
 
         let mut data = String::new();
         file.read_to_string(&mut data)
-            .map_err(|error| anyhow::anyhow!("Failed to read test file: {}", error))?;
+            .map_err(|error| anyhow::anyhow!("Failed to read test file (1): {}", error))?;
```

Comment:
@FlashSheridan , what is the `(1)` meaning here? I believe it will be just printed out as a string, without any expansion.

Reply:
> ‘Disambiguate 3 × "Failed to read test file" msgs.’

When I was investigating an error with this message string, it wasn’t clear which of the three occurrences it was.   (It turned out to be the one I labelled “(2).”)  I didn’t know the code well enough to make the three differ by including more detail, so I just added numbers.  (In my own code I’ve sometimes tweaked capitalization or punctuation; open to better suggestions.)